articles/best-practices/connection-settings.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Recommended identity provider connection settings in Auth0.
+topics:
+  - best-practices
+  - configuration
+  - settings
+  - connection
+contentType: 
+    - index
+    - reference
+useCase:
+  - best-practices
+  - connection
+  - connection-settings
+---
+
+# Recommended Connection Settings
+
+In this article, you'll learn some best practices for configuring [connections](/connections). Before you set up connections, take a moment to review [what connections are](/applications/connections) and [the basics of authentication](/application-auth) for your application type.
+
+## Use your credentials for social connections
+
+Auth0 provides [default credentials](/connections/social/devkeys) for [social connections](/social/connections) to help you get started. You should replace these temporary credentials with your own to avoid restrictions.
```

Comment:
/social/connections is broken - do you mean it to go here? https://auth0.com/docs/identityproviders#social